src/styles/sidebar/components/styled-text.scss

```diff
@@ -0,0 +1,85 @@
+/**
+ * Apply styling to a block of text.
+ *
+ * The `:where` pseudo-class always has a specificity of 0, which allows us
+ * to select for elements while maintaining the same specificity of a single
+ * class name throughout.
```

Comment:
Support for `:where` requires more recent browsers than our current baseline. See https://caniuse.com/?search=%3Awhere (Chrome >= 88, Firefox >= 78, Safari >= 14). Is there a fallback for older browsers, down to those specified in the project's `package.json`?

Reply:
I notice in the docs for the related [:is](https://developer.mozilla.org/en-US/docs/Web/CSS/:is#browser_compatibility) pseudo-class that `:is` has been supported for a long time under vendor prefixes (`:-webkit-any`, `:-moz-any`). It might be sufficient to use that as a fallback. This fallback would I believe have the same specificity as `:is` rather than `:where`, so it depends if that is acceptable for this use case.